### DISCUSSION TITLE
bricked by Vysor?

### DISCUSSION THREAD

**Discussion by captracks** [makerdiary/nrf52840-mdk-usb-dongle#108] [Q&A]
I just picked up a makerdiary nRF52840 to use with Vysor as a dongle per this doc: https://github.com/koush/vysor.io/wiki/Vysor-Dongle#nrf52840-flashing-instructions

It was recognized as a drive when I rest it and connected to my ubuntu workstation but when I dropped the image into it I got an error and it disconnected.  Now if I try to reset it by holding down the reset button while inserting or clicking it it just sits there with a green power LED.  Is it bricked?

**Comment by jarecto** [comment]
I'm in the same boat, and would appreciate any guidance on this. In my case the LED no longer powers on - it was able to be recognized as a Vysor Dongle right after flashing, but no longer once I unplugged it and plugged it back in.
